llvm/include/llvm/CGData/CGDataPatchItem.h

```diff
@@ -22,10 +22,10 @@ struct CGDataPatchItem {
   // Where to patch.
   uint64_t Pos;
   // Source data.
-  OwningArrayRef<uint64_t> D;
+  std::vector<uint64_t> D;
```

Comment:
also here

Reply:
I haven't audited all of the uses of `CGDataPatchItem` to ensure that none of the clients rely on a reference to the data being stable on a move. That is guaranteed by `OwningArrayRef` and `std::vector` but not `SmallVector`.